packages/example-next-13-with-pages/README.md

```diff
@@ -0,0 +1,20 @@
+# example-next-13-with-pages
+
+An example that showcases a basic installation of `use-intl` in a Next.js app that uses both the pages and the new app folder.
+
+Important to note:
+
+1. Set up your project according to the official [docs for Next.js 13](https://next-intl-docs.vercel.app/docs/next-13)
+2. Do not add the i18n config to the `next.config.js` as it does not work with the new `app` folder and the `pages` folder.
```

Comment:
Maybe this could be a bit more specific, e.g. "as it will cause the `app` directory to be ignored.".